### PR TITLE
🌐 N°7241 - Update czech translations

### PR DESCRIPTION
**Pull request by Stetinac**

### CODE REVIEW

File: cs.dict.combodo-approval-extended.php [hunk 1]
```diff
@@ -95,7 +95,7 @@
 	'Class:ApprovalRule/Attribute:level2_substitute_query+' => 'Substitutes are approver dependent : use `:approver->...` placeholder in the query to retrieve the corresponding substitutes~~',
 	'Class:ApprovalRule/Attribute:level2_substitute_timeout' => 'Substitute notification delay L2~~',
 	'Class:ApprovalRule/Attribute:level2_substitute_timeout+' => 'Substitutes will be notified if approver has not answered before this percentage of the approval delay~~',
-	'Class:ApprovalRule/Attribute:servicesubcategory_list' => 'Podkategorie služeb',
+	'Class:ApprovalRule/Attribute:servicesubcategory_list' => 'Service subcategory~~',
```
Comment (piRGoif) [inline cs.dict.combodo-approval-extended.php:98]: Is setting back the english version of this key really intended ?

**Review by piRGoif — CHANGES_REQUESTED** [review]

File: cs.dict.combodo-approval-extended.php (hunk 1 above)
Comment (Stetinac) [reply cs.dict.combodo-approval-extended.php:98]: This is an error from our internal merging scripts :-(
The previous translation is correct.

Comment (Molkobain) [inline cs.dict.combodo-approval-extended.php:98]:
```suggestion
	'Class:ApprovalRule/Attribute:servicesubcategory_list' => 'Podkategorie služeb',
```

**Review by Molkobain — APPROVED** [review]

**Review by piRGoif — APPROVED** [review]

**Comment by Molkobain** [comment]
Accepted during technical review

**Comment by Molkobain** [comment]
Accepted during functional review.

**Pull request merged**